### DISCUSSION TITLE
CrowdSec Access Forbidden

### DISCUSSION THREAD

**Discussion by Matt-CyberGuy** [ZoeyVid/NPMplus#896] [Q&A] [answered]
Love this project, especially with QUIC enabled out of the box for it, but I'm struggling to get the crowdsec portion working. I've followed the instructions, but when I enable crowdsec, I just get 'CrowdSec Access Forbidden' on every port. My ip is blocked... how do I whitelist it or get crowdsec to function properly?

<img width="706" alt="image" src="https://github.com/ZoeyVid/NPMplus/assets/65490814/561435ae-dc10-404b-90d8-5efdb235c1c3">

**Comment by Matt-CyberGuy** [comment]
Also, all the instances I have spun up have recently started showing up in portainer as unhealthy. I'm not running these in host mode, mostly so it can be accessed by it's hostname by my cloudflared tunnel, and so npm can access my running containers by hostname

![image](https://github.com/ZoeyVid/NPMplus/assets/65490814/c8cf9923-d784-44f2-bb1e-8a72df3ea7e0)

**Comment by Zoey2936** [comment, marked as answer]
since you don't use network mode host, crowdsec and npmplus need to be in the same network and you need to change all "127.0.0.1" to the hostname/ip of the crowdsec container in /opt/npm/etc/crowdsec/crowdsec.conf

**Comment by Matt-CyberGuy** [comment]
Thanks, I think I may have figured it out. After adding recaptcha, putting in the site key and secret, crowdsec seem to start working. Now I just gotta figure out how to test to see if it'll block stuff
